src/ape_ethereum/ecosystem.py

```diff
@@ -147,6 +168,89 @@ def decode_address(cls, raw_address: RawAddress) -> AddressType:
     def encode_address(cls, address: AddressType) -> RawAddress:
         return str(address)
 
+    def is_proxy(self, address: AddressType) -> bool:
+        return self.get_proxy_info(address) is not None
```

Comment:
Maybe we don't need this afterall w/ `Optional`, can just do `.get_proxy_info(address).target or address`

Reply:
.target will fail if it returns None